### PR TITLE
chore: Initial release setup

### PR DESCRIPTION
**Pull request by magnuh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,9 +15,12 @@
     "#dev-util": "./dev/src/util.js"
   },
   "scripts": {
+    "commit": "cz",
     "cp": "mkdir -p ./dev/public/shiki && cp -r ./node_modules/shiki/{dist,languages,samples,themes} ./dev/public/shiki",
     "cp:shikifix": "cp node_modules/.pnpm/vscode-oniguruma@1.7.0/node_modules/vscode-oniguruma/release/onig.wasm ./dev/public/shiki/dist/",
     "dev": "vite dev dev --config ./vite.config.js",
+    "eik:login": "npx @eik/cli login",
```
Comment (pearofducks) [inline package.json:22]: I'd prefer `@eik/cli` as a devDep instead and running this directly via `eik login` instead of `npx @eik/cli login` (same with the publish step).

Comment (magnuh) [reply package.json]: Fixed 👍

**Review by pearofducks — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -73,5 +81,14 @@
...
-  ]
+  ],
+  "config": {
+    "commitizen": {
+      "path": "./node_modules/cz-conventional-changelog"
+    }
+  },
+  "eik": {
+    "server": "https://assets.finn.no",
+    "files": "./dist"
```
Comment (BalbinaK) [inline package.json:92]: Did we test that we need to specify the files here if the property already exists in line 83?

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        with:
+          version: 7
+          run_install: true
+      - name: Build
+        run: pnpm build
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npx semantic-release
```
Comment (BalbinaK) [inline .github/workflows/release.yml:31]: I see you added logic for publishing to eik in package.json, but you haven't added the eik step here - are you planning to do it in a separate PR?

**Review by BalbinaK — COMMENTED** [review]
Nice! Just a few things I wanted to point out 😊

File: package.json (hunk 2 above)
Comment (AnnaRybkina) [reply package.json:90]: I want to remember that "files" are not inherited. Only `name` and `version`.

Comment (AnnaRybkina) [reply package.json:90]: Can be wrong ofc 😅

File: package.json [hunk 4]
```diff
@@ -14,24 +15,23 @@
...
     "build": "rm -rf dist && rollup -c",
-    "publish:eik": "npx @eik/cli publish && npx @eik/cli pkg-alias",
+    "publish:eik": "eik publish && eik pkg-alias",
     "site": "vite build dev --config ./vite.config.js",
     "postsite": "cd dev/dist && cp index.html 200.html && cp index.html 404.html && cp ../CNAME .",
-    "preversion": "npm run test && npx @eik/cli login && npm run build",
+    "preversion": "pnpm run test && eik login && pnpm run build",
     "version": "npm publish",
-    "postversion": "npm run publish:eik && git push --follow-tags"
+    "postversion": "pnpm run publish:eik && git push --follow-tags"
```
Comment (AnnaRybkina) [inline package.json:30]: Do we need those now? and preversion

```suggestion
    "postversion": "pnpm run publish:eik && git push --follow-tags"
```

File: package.json (hunk 2 above)
Comment (magnuh) [reply package.json:90]: I guess it's needed only in case we don't want the same as in the base `files` entry...? Haven't looked how it is set up in the other repos... but I think I like to have it there always (for consistency and clarity)... What do you think?

Comment (magnuh) [reply package.json:90]: We kept it in release-playground... so I guess that was for some reason? 😄

File: package.json (hunk 4 above)
Comment (magnuh) [reply package.json:30]: Probably not? I kept them because I didn't know at this point. 😄  ...but perhaps we should just delete them, and re-introduce from Fabric if needed?

Comment (AnnaRybkina) [reply package.json:30]: We do have another publishing procedure so this can be removed.

File: package.json (hunk 2 above)
Comment (AnnaRybkina) [reply package.json:90]: As I've mentioned previously we need `files` as it is only `name` and `version` from package.json that will be used if omitted in the eik config. If you remove files from eik you'll get 👇 
image

**Review by AnnaRybkina — APPROVED** [review]
